example_app_generator/generate_app.rb

```diff
@@ -40,6 +40,9 @@
   end
 
   if Rails::VERSION::STRING >= "5.1.0"
+    if RUBY_VERSION < "2.4"
+      gsub_file "Gemfile", /.*capybara.*/, "gem 'capybara', '~> 3.10.0'"
```

Comment:
We can maybe use the last version before 2.4 limitation https://github.com/teamcapybara/capybara/blob/master/History.md#changed

`3.15.0` instead of `3.10.0.

This is not a blocker

Reply:
👍